node/lib/ILogger.ts

```diff
@@ -1,8 +1,11 @@
 export interface ILogger {
-    error(msg: string, logObject?: any): void;
-    warn(msg: string, logObject?: any): void;
-    info(msg: string, logObject?: any): void;
-    verbose(msg: string, logObject?: any): void;
-    debug(msg: string, logObject?: any): void;
+    // Example of loggingMetadata is correlationVector
+    error(msg: string, logObject?: any, loggingMetadata?: string): void; 
```

Comment:
example in opent2ttests.ts

---
In reply to: [105465737](https://github.com/openT2T/opent2t/pull/38#discussion_r105465737) [](ancestors = 105465737)